tiledb/sm/config/config.cc

```diff
@@ -91,6 +91,7 @@ const std::string Config::REST_RETRY_DELAY_FACTOR = "1.25";
 const std::string Config::REST_CURL_BUFFER_SIZE = "524288";
 const std::string Config::REST_CAPNP_TRAVERSAL_LIMIT = "2147483648";
 const std::string Config::REST_CURL_VERBOSE = "false";
+const std::string REST_CURL_TCP_KEEPALIVE = "true";
```

Comment:
```suggestion
const std::string Config::REST_CURL_TCP_KEEPALIVE = "true";
```

Reply:
I intentionally did it this way to avoid changing another file (`config.h`) for no reason; see also line 178. We should make all declarations private to `config.cc` in a subsequent PR.

Reply:
It's probably good to keep these variables namespaced like the existing config parameters used here.

Reply:
Done.